pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/kdefault/manager_auth_proxy_patch.go

```diff
@@ -53,22 +53,6 @@ metadata:
 spec:
   template:
     spec:
-      affinity:
```

Comment:
@NikhilSharmaWe, if we will remove then we need to remove it from kustomize v1 and kustomize v2.
Could you please change it in both?

Reply:
Fixed